manticore/ethereum.py

```diff
@@ -448,6 +443,17 @@ def __str__(self):
     def address(self):
         return self._address
 
+    def _init_hashes(self):
+        #initializes self._hashes lazy
+        if self._hashes is None and self._seth is not None:
+            self._hashes = {}
+            md = self._seth.get_metadata(self._address)
+            if md is not None:
```

Comment:
_init_hashes will take effect only if metadata is defined. 
It may happen (happened before) that the account is created but the metadata is not yet set. So we need to delay the hashes initialization for later. ?